bytebuffer-collections/src/main/java/io/druid/collections/spatial/RTree.java

```diff
@@ -128,8 +128,8 @@ private Node buildRoot(boolean isLeaf)
   {
     float[] initMinCoords = new float[numDims];
     float[] initMaxCoords = new float[numDims];
-    Arrays.fill(initMinCoords, -Float.MAX_VALUE);
-    Arrays.fill(initMaxCoords, Float.MAX_VALUE);
+    Arrays.fill(initMinCoords, Float.NEGATIVE_INFINITY);
```

Comment:
I am not familiar with the code of Rtree but am not sure if this change really make sense. After this change any operation on the root node coordinate will always return infinity, while before i can have for instance an incremental change over the root coordinate. 
Eg,  ``` node.getMinCoordinates()[0] [* or /] 100```  will return `Infinity` if we use `Float.NEGATIVE_INFINITY` and it will have a defined value if we keep `Float.MAX_VALUE`